samples/cellular/udp/src/main.c

```diff
@@ -25,7 +25,6 @@ static void socket_transmission_work_fn(struct k_work *work)
 {
 	int err;
 	char buffer[CONFIG_UDP_DATA_UPLOAD_SIZE_BYTES] = {"\0"};
-	int rai;
```

Comment:
I'd like to keep the declaration of the top of the function, though I understand that this is moved as the variable is not always needed and there is no single Kconfig to check whether `rai` should be declared.